app/brave_main_delegate.cc

```diff
@@ -161,9 +162,12 @@ void BraveMainDelegate::PreSandboxStartup() {
 
 bool BraveMainDelegate::BasicStartupComplete(int* exit_code) {
   BraveCommandLineHelper command_line(base::CommandLine::ForCurrentProcess());
-#if BUILDFLAG(BRAVE_ADS_ENABLED)
+#if BUILDFLAG(BRAVE_ADS_ENABLED) || BUILDFLAG(ENABLE_SPEEDREADER)
   command_line.AppendSwitch(switches::kEnableDomDistiller);
-#endif
+#endif  // BUILDFLAG(BRAVE_ADS_ENABLED) || BUILDFLAG(ENABLE_SPEEDREADER)
+#if BUILDFLAG(ENABLE_SPEEDREADER)
+  command_line.AppendSwitch(switches::kEnableDistillabilityService);
```

Comment:
are you sure adding chrome distiller should be part of this PR? wouldn't it be rational to start with our machinery first and then consider adding chrome distiller?

Reply:
it's only there to show the reader mode button. it's part of the UI spec.

Reply:
It seems I don't fully understand the spec :) So what is called "normal reader mode" is basically having a page distilled by the chrome distiller?

Reply:
No. I use the Soeedreader throttle. The tab helper sets SingleShotMode. It creates the throttle for the next request and calls reload.

Reply:
But how chrome distiller is related to speedreader? If the observer shows that the page is distillable, is it guaranteed that speedreader will optimize it? I'm confused with the fact that we rely on Chromium distiller. Also performance implications are not really clear.

Reply:
I'd rather don't rely on chrome distiller, as discussed on Slack

Reply:
let's just try it out because it helps for the long tail. we have until July 13th